classes/class-wc-connect-order-presenter.php

```diff
@@ -0,0 +1,153 @@
+<?php
+
+if ( ! class_exists( 'WC_Connect_Order_Presenter' ) ) {
+
+	class WC_Connect_Order_Presenter {
+
+		/**
+		 * This function transform the WC_Order object to a representational JSON form for the react app.
+		 * This is based on WooCommerce v3's get_order API woocommerce/includes/legacy/api/v3/class-wc-api-orders.php
+		 *
+		 * @param WC_Order $order
+		 * @return array
+		 */
+		public function get_order_for_api( WC_Order $order ) {
+			$dp = 2; //decimal point defaults
```

Comment:
(minor / ⛏️ ) Instead of `$dp` maybe expand to `$decimal_points`? By expanding the acronym, then you can also save the comment that explains what `$dp` stands for 😄

Reply:
Updated!